app/hotels/src/filter/price/PricePopup.js

```diff
@@ -77,30 +78,35 @@ export default class PricePopup extends React.Component<Props, State> {
           onClose={this.props.onClose}
           isVisible={this.props.isVisible}
         >
-          <Text style={styles.title}>
-            <Translation id="hotels_search.filter.price_popup.title" />
-            <Translation passThrough=" " />
-            <Text style={styles.subtitle}>
-              <Translation id="hotels_search.filter.price_popup.subtitle" />
+          <View style={styles.titleContainer}>
+            <Text style={styles.title}>
+              <Translation id="hotels_search.filter.price_popup.title" />
+              <Translation passThrough=" " />
             </Text>
-          </Text>
-          <SliderLabels
-            max={max}
-            min={min}
-            startLabel={
-              <Price amount={start * daysOfStay} currency={currency} />
-            }
-            startValue={start}
-            endLabel={<Price amount={end * daysOfStay} currency={currency} />}
-            endValue={end}
-          />
+            <AdaptableBadge
+              translation={
+                <TranslationFragment>
+                  <Price amount={start * daysOfStay} currency={currency} />
```

Comment:
Should we use the number library to compute the price?

Reply:
You mean decimal? No, since this particular value still comes as Float from the server.